apps/web/components/Wrapper.js

```diff
@@ -4,14 +4,16 @@ import * as React from 'react';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 import styled from 'styled-components';
 import { LAYOUT } from '@kiwicom/margarita-utils';
+import { View } from 'react-native';
 
 import Layout from './Layout';
 
 type Props = {|
   +children: React.Node,
 |};
 
-const ChildrenWrapper = styled.div({
+const ChildrenWrapper = styled(View)({
+  flex: 1,
   marginTop: defaultTokens.spaceXSmall,
```

Comment:
Can you please just switch margin values to padding. Because content is currently scrolling just within area with margin. And content is cut on top because of top margin.

<img width="1440" alt="screenshot 2019-02-25 at 14 46 40" src="https://user-images.githubusercontent.com/2660330/53341623-5467f280-390c-11e9-87ea-8a12c45bd7f7.png">

Reply:
This approach will not move the scroll-bar out to the right even if we use padding, because the scroll-bar is attached to the div resulting from the ScrollView

Reply:
We might need to move the `ScrollView` out of `core` and wrap `web` and `mobile` with the scroll view instead. I will check it

Reply:
Right, it worked with div but in this case the structure is slightly different.

We probably need to add extra View for example like this.
![screenshot 2019-02-25 at 15 13 30](https://user-images.githubusercontent.com/2660330/53343203-153ba080-3910-11e9-8dd7-66c782b96089.png)

Reply:
The problem with that approach is that you are talking the scroll responsability away from the Scroll view and moving int to the `Container` component.

Reply:
👍 Right, I didn't noticed that there is `ScrollView` inside.